.changeset/nasty-readers-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/perseus-editor": minor
+---
+
+Rollback popover direction removal from label image editor and fix bug
```

Comment:
I might update this to be a bit more clear on what bug was fixed. Maybe "Add popover direction selector back to label image editor and fix bug where the selected choice was not persisting"?